README.md

```diff
@@ -240,7 +240,13 @@ moodle-docker-compose exec webserver docker-php-ext-enable xdebug
 moodle-docker-compose restart webserver
 ```
 
-While setting these XDebug settings depending on your local need, please take special care of the value of `xdebug.client_host` which is needed to connect from the container to the host. The given value `host.docker.internal` is a special DNS name for this purpose within Docker for Windows and Docker for Mac. If you are running on another Docker environment, you might want to try the value `localhost` instead or even set the hostname/IP of the host directly.
+While setting these XDebug settings depending on your local need, please take special care of the value of `xdebug.client_host` which is needed to connect from the container to the host. The given value `host.docker.internal` is a special DNS name for this purpose within Docker for Windows and Docker for Mac. If you are running on another Docker environment, you might want to try the value `localhost` instead or even set the hostname/IP of the host directly. Please turn off the firewall or open the port used in the xdebug.client_port.
```

Comment:
Suggestion: add the backticks (`) around `xdebug.client_port`.

HTH,
Matteo

Reply:
Hi @scara . I'm sorry for the late response. I had wrapped `xdebug.client_port` with the backticks.